django/contrib/postgres/serializer.py

```diff
@@ -0,0 +1,6 @@
+from django.db.migrations.serializer import BaseSerializer
+
+
+class DateTimeTZRangeSerializer(BaseSerializer):
+    def serialize(self):
+        return "%s" % repr(self.value), {}
```

Comment:
Use single quotes (unless a string contains a single quote) as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style). (Please check the entire patch for this.)